system/jlib/jmisc.hpp

```diff
@@ -22,6 +22,8 @@
 
 #include "platform.h"
 #include <stdio.h>
+#include <string>
```

Comment:
because needed by the prototypes:
```
extern jlib_decl std::pair<std::string, unsigned> getExternalService(const char *serviceName);
extern jlib_decl std::pair<std::string, unsigned> getDafileServiceFromConfig(const char *application);

```